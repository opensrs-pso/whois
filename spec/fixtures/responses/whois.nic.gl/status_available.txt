
Domain Information
Query: u34jedzcq.gl
Status: Not Registered



