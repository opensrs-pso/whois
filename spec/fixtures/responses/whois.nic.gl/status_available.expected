#domain
  should: %s == "u34jedzcq.gl"

#domain_id
  should: %s raise_error(Whois::AttributeNotSupported)


#status
  should: %s == :available

#available?
  should: %s == true

#registered?
  should: %s == false


#created_on
  should: %s == nil

#updated_on
  should: %s == nil

#expires_on
  should: %s == nil


#registrar
  should: %s == nil


#nameservers
  should: %s CLASS(array)
  should: %s == []
